crates/bevy_pbr/src/material.rs

```diff
@@ -370,7 +370,7 @@ impl<P: PhaseItem, M: Material, const I: usize> RenderCommand<P> for SetMaterial
     }
 }
 
-pub type RenderMaterialInstances<M> = RenderInstances<AssetId<M>>;
+pub type RenderMaterialInstances<M> = Extracted<AssetId<M>>;
```

Comment:
Am I correct, this type is used only for rendering purposes? In this case naming `RenderInstances` feels more proper than `Extracted` 🤔